src/test/java/org/manifold/compiler/front/TestPrimitiveNodeVertex.java

```diff
@@ -339,14 +317,7 @@ public void testElaborate_outputNil()
     ExpressionEdge eNodePorts = new ExpressionEdge(vNodePorts, null);
     g.addEdge(eNodePorts);
     
-    ConstantValueVertex vAttributes = new ConstantValueVertex(g,
-        NilTypeValue.getInstance());
-    ExpressionEdge eAttributes = new ExpressionEdge(vAttributes, null);
-    g.addVertex(vAttributes);
-    g.addEdge(eAttributes);
-    
-    PrimitiveNodeVertex vNode = new PrimitiveNodeVertex(g,
-        eNodePorts, eAttributes);
+    PrimitiveNodeVertex vNode = new PrimitiveNodeVertex(g, eNodePorts);
     g.addVertex(vNode);
     
     vNode.elaborate();
```

Comment:
No, I'm working on the test cases in a different branch so that this wouldn't be 2000+ lines